### ISSUE TITLE
Can't open tun devices

### ISSUE THREAD

**Issue by Red-M** [danfruehauf/NetworkManager-ssh#77]
Maybe I am doing this wrong but I can't seem to get any tun devices opened at all even if I am in the correct groups that should be able to open a tun interface.

This could be a distro issue as I did have this working without root on both sides before, however I have had to rebuild a few machines lately due to other issues.

**Comment by danfruehauf** [comment]
> This could be a distro issue as I did have this working without root on both sides before, however I have had to rebuild a few machines lately due to other issues.

If you got it to work without root, let me know what you did. But generally you need root to open new tun devices...

**Comment by Red-M** [comment]
I was in the `netdev` group on both machines however it seems an update may have reverted this.

Can I ask that some extra options to allow for sudo/su on the requesting/client machine be added so that I may supply a password to allow the tun on my machine to be opened?

**Comment by danfruehauf** [comment]
> Can I ask that some extra options to allow for sudo/su on the requesting/client machine be added so that I may supply a password to allow the tun on my machine to be opened?

Yes, it is a very fair ask. I've been wondering for a while what's the best way to go about that - so there is flexibility with the remote command that is going to be executed. I'm open to ideas that can make it easy for more people.
